src/Tribe/Aggregator/API/Origins.php

```diff
@@ -232,7 +232,7 @@ public function get_name( $id ) {
 		$this->get();
 
 		if ( empty( $this->origins[ $id ] ) ) {
-			return __( 'Event Aggregator', 'the-events-calendar' );
```

Comment:
I think making these not translatable is a move in the wrong direction. By having them translatable, the translator gets to choose how they should appear and respect whatever is most appropriate for their language. There isn't a great justification for removing that choice.

Reply:
@zbtirrell We talked about this in the #products-i18n channel and on scrum a bit already. The point is that these are trademarks and should not be altered—thoughts? Specific exception is _Google Calendar_, because they translate "Calendar" themselves, but we do not. "Meetup" is just "Meetup", meanwhile, all of the time. Ditto "The Events Calendar", "Eventbrite", "Event Aggregator", etc... Right?